### DISCUSSION TITLE
</> Actions

### DISCUSSION THREAD

**Discussion by Code1109** [codefori/vscode-ibmi#579] [Ideas]
I added a variant of CRTSQLRPGI compile command for when compiling directly on the IBMi (not the IFS). 
Happy to share with y'all:

--MODULE--
?CRTSQLRPGI OBJ(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) OBJTYPE(*MODULE) CLOSQLCSR(*ENDMOD) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)

--PROGRAM--
?CRTSQLRPGI OBJ(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) CLOSQLCSR(*ENDMOD) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)

Just create the actions on the 'work with actions' section at the bottom there's a button for [new action] or you can also [duplicate] and modify as needed.

**Comment by worksofliam** [comment]
Thanks for sharing!!
